xapi-profiles-communication.md

```diff
@@ -23,39 +23,75 @@ Part Two:	[xAPI Profiles Document Structure Specification](./xapi-profiles-struc
+offer separate querying on them. One of these is the default graph, which is queried 
+when no other graph is specified. The default graph at this SPARQL endpoint will 
+contain all the current versions of profiles. 
+
+### <a name="profile_versions">1.1</a> Profile Versions
+
+Every profile version will be in a [Named Graph](https://www.w3.org/TR/sparql11-query/#namedGraphs) 
+with a URI equal to the profile version's URI.  Non-current (and current) versions of 
+Profiles are required to be in named graphs with URIs of their version.  This means 
+they can be queried directly by specifying the URI. Additionally, the current version 
+in the default graph has the versioning information, making it possible to traverse 
+them without knowing their URIs in advance. Reasons why one would query versions 
+of xAPI Profiles include retrieving a version used by an xAPI Statement, and/or 
+understanding when a concept was deprecated or removed.
 
-The review process will check profiles for following the specification and assist in helping them be of highest quality, after which they will be added to the server.
+The URI of the named graph is always the URI of the profile version. Note: The URI of the 
+named graph doesn't mean the URI it is retrieved at, it means the URI used to refer to it 
+within SPARQL/the RDF store.
 
-A Profile Server will host a SPARQL endpoint containing the RDF information from the contained profiles at the path /SPARQL. SPARQL servers have the ability to divide information into multiple datasets, or graphs, and offer separate querying on them. One of these is the default graph, which is queried when no other graph is specified. The default graph at this SPARQL endpoint will contain all the current versions of profiles. Additionally, every profile version will be in a Named Graph with a URI equal to the profile version's URI. Thus, by default queries will only operate on up to date information, but if historical profile information is needed, it is available.
+In summary: by default, queries will only operate on up to date 
+information; if historical profile information is needed, it is available.
 
-A Profile Server will include inference logic for the following, at minimum: all SKOS predicate relationships, and `profile:concepts`, `profile:templates`, and `profile:patterns` being subproperties of the inverse of `skos:inScheme`.
+### <a name="profile_server_best_practices">1.2</a> Best Practices
 
+Some best practices are recomended when adding a profile to a profile server. A technical 
+review process is warranted to check profiles for following the specification. As well, 
+analysis is recommended to ensure the contents of a profile are of the highest quality. 
+Upon following these practices, a profile should be added to a profile server.
 
-### <a name="1.1">1.1</a> Example SPARQL Queries
+As language maps enable the multiple translations of an xAPI Profile, it is advisable to
```

Comment:
It doesn't really have anything to do with language maps (those are just a syntactic representation of the underlying data), so I'd probably just rephrase this to something like "As Profiles support localizations in multiple languages and localities"